cves/CVE-2013-0848.yml

```diff
@@ -361,11 +371,11 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
+CWE: CWE-119
 CWE_note:
-nickname_instructions: |
+nickname_instructions:
   A catchy name for this vulnerability that would draw attention it. If the
   report mentions a nickname, e.g. "Heartbleed", use that. Or come up with one!
 
   Must be under 30 characters. Optional. Be appropriate.
-nickname:
+nickname: Out-of-Bounds more like Get out-of town
```

Comment:
You're more creative than me, nice! I'll come back and give some upvotes once I'm doing reading the rest 👍

Reply:
+1 upvote

Large impact from a fairly routine mistake.